CHANGES.rst

```diff
@@ -10,6 +10,9 @@ New Features
 
 - Data color cycler and marker color updates for increased accessibility. [#2453]
 
+- Add support for ``MultiMaskSubsetState`` in ``viz.app.get_subsets()`` and in
+  the Subset Plugin [#2463]
```

Comment:
This is why its failing changelog check (wrong PR number)

```suggestion
  the Subset Plugin [#2462]
```

Reply:
🤦

Reply:
Thank you